### ISSUE TITLE
FirebaseUI install via Cocoapods is linking libraries again

### ISSUE THREAD

**Issue by netdawgx** [firebase/FirebaseUI-iOS#123]
FirebaseUI install via Cocoapods is linking libraries again, this causes a huge amount of errors when the project is built (still builds successfully though). I have removed pods 1 by 1 and built the project and FirebaseUI seems to be the culprit.

There are many errors but of the same type, here is an example:

objc[594]: Class FIRAAppEnvironmentUtil is implemented in both /private/var/containers/Bundle/Application/F68199FC-1151-432B-8BF2-6D2113588D13/hotspotz.app/Frameworks/FirebaseGoogleAuthUI.framework/FirebaseGoogleAuthUI and /private/var/containers/Bundle/Application/F68199FC-1151-432B-8BF2-6D2113588D13/hotspotz.app/Frameworks/FirebaseFacebookAuthUI.framework/FirebaseFacebookAuthUI. One of the two will be used. Which one is undefined.

I have the podfile set up to use frameworks. I am using Swift, XCode 7.3
Any ideas on how to stop the errors from happening?

Here is a copy of my podfile:

```
 # Uncomment this line to define a global platform for your project
 platform :ios, '9.0'

target 'hotspotz' do
  # Comment this line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for hotspotz
pod 'Firebase/Core'
pod ‘Firebase/Database’
pod 'GeoFire', :git => 'https://github.com/firebase/geofire-objc.git'
pod 'FirebaseUI', '~> 0.5'
pod 'Eureka', '~> 1.7'
end
```

**Comment by nhantrivinh** [comment]
same

**Comment by morganchen12** [comment]
related to #121, #124

**Comment by morganchen12** [comment]
This is fixed in 0.5.3.
